starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw

```diff
@@ -159,6 +159,15 @@ azurefunctions {
 }
 }
 
+@if (features.contains("aws-lambda")) {
```

Comment:
this should only be present if `graalvm` feature is present as well ?

Reply:
What about the equivalent configuration for Maven users?

Reply:
Note that today the experience is for new users that if they just choose the defaults and create an AWS lambda app it by default will not be able to run (fails on the 3rd or 4th request) unless these memory settings are in place.

If we are happy for that to be the developer experience for Lambda users then we can close this PR

Reply:
@sdelamo just pinging that the other PR you created has test failures, are we going to leave this as is for longer?

Reply:
Let me check the other PR.